File: ADT102/ThirdAssignment/Student.java
package com.company;

public class Student {
    private String studentID;
    private String studentName;
    private String title;
    private String studentLastname;

    public Student(String studentID,String title,String studentName, String studentLastname) {
        this.studentID = studentID;
        this.studentName = studentName;
        this.title = title;
        this.studentLastname = studentLastname;
    }

    @Override
    public String toString() {
        return "Student{" +
                "ID='" + studentID + '\'' +
                ", title='" + title + '\'' +
                ", Name='" + studentName + '\'' +
                ", Lastname='" + studentLastname + '\'' +
                '}';
    }
}

File: ADT102/pythonAssign/Student.py
class Student:
    def __init__(self, st_id, name, last_name):
        self.st_id = st_id
        self.name = name
        self.last_name = last_name

    def __str__(self):
        return "id:", self.st_id, "name:", self.name, "lastname:", self.last_name
